anchore_engine/services/catalog/api/controllers/default_controller.py

```diff
@@ -196,6 +195,27 @@ def get_image(imageDigest):
     return return_object, httpcode
 
 
+@flask_metrics.do_not_track()
+@authorizer.requires_account(with_types=INTERNAL_SERVICE_ALLOWED)
+def get_image_content(image_digest, content_type):
+    return_object = {}
+    httpcode = 500
+
+    try:
+        image_content_data = CatalogService.get_image_content(
```

Comment:
@zhill , is there a way to lookup the service dynamically rather than using a class method here? I'd like to make this compatible with any extensions of the Catalog Service